source/reference/command/find.txt

```diff
@@ -74,6 +74,11 @@ The :dbcommand:`find` command has the following syntax:
       }
    )
 
+.. note::
+  Since MongoDB 4.4, the :dbcommand:`find` command ignores the
+  ``oplogReplay`` flag. However, if the ``oplogReplay`` flag is set, 
+  the server accepts it for backwards compatibility, but has no effect.
+  
```

Comment:
I'll defer to external review on this, but I'm wondering if we even need this note. Since we've removed all other docs for this command I'm wondering if calling this out will just be more confusing than helpful to users. Just feels slightly odd to me to call out this flag with no explanation of what it is / does.